code/datums/gamemode/factions/bloodcult/bloodcult_buildings.dm

```diff
@@ -728,6 +728,9 @@
 		newCultist.conversion.Add("altar")
 		cult_risk()//risk of exposing the cult early if too many soul blades created
 
+		spawn(1)
```

Comment:
Eww
Is there really no other way to do this ?

Reply:
By all logic there's no reason this should create braindead shades, yet it does, so this is an attempt to bruteforce the ghost into the shade.

Reply:
How about this : 
```
if (!usr.client)
    stack_trace("We wanted to make a shade, but there is no ckey!")
    spawn(1)
       shademob.ckey = M_ckey
else
    shademob.ckey = usr.ckey
```
That way it fixes the bug but also leaves a helpful runtime if we want to know the root cause later on

Reply:
you mean `if (!shademob.client)` ?

Reply:
No, I mean usr. If the shade has a client I **believe** it will always have a ckey.

Might need to test it.